src/pages/signup/styles/signup.css

```
@@ -0,0 +1,247 @@
+}
+
+.signup-form {
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+}
+
+/* Title */
+.signup-title {
+  color: #313131;
+  font-family: "Pretendard", system-ui, -apple-system, "Segoe UI", Roboto, Arial, sans-serif;
```

Stylelint (17.3.0)

[error] Unexpected quotes around "Pretendard" (font-family-name-quotes)

(font-family-name-quotes)